lib/src/model/movies_model.dart

```diff
@@ -0,0 +1,37 @@
+import 'movies_result_model.dart';
+
+class MoviesModel {
```

Comment:
these file and class are already on the model folder, you should rename it to Movie or something similar

Reply:
done